services/ui-src/src/measures/2023/rateLabelText.ts

```diff
@@ -22,6 +22,16 @@ export const data = {
         ],
         "categories": [{"id":"SU6HXz", "label": "", "text":""}]
     },
+    "AAB-CH": {
+        "qualifiers": [
+            {
+                "label": "Ages 3 months to 17 years",
+                "text": "Ages 3 months to 17 years",
+                "id": "xS5HMm"
+            },
+        ],
+        "categories": []
```

Comment:
```suggestion
        "categories": [{"id":"ZCy3XP", "label": "", "text":""}]
```

slight change needed due to the label update, all categories need to have an id